src/Microsoft.AspNet.OData.Shared/Builder/CollectionPropertyConfiguration.cs

```diff
@@ -63,5 +63,14 @@ public CollectionPropertyConfiguration IsRequired()
             OptionalProperty = false;
             return this;
         }
+
+        /// <summary>
+        /// Adds TSubClass to the list of derived type constraints.
+        /// </summary>
+        public CollectionPropertyConfiguration AddDerivedTypeConstraint<TSubClass>() where TSubClass : class
+        {
+            this.AddDerivedTypeConstraintImpl(typeof(TSubClass));
```

Comment:
>this [](start = 12, length = 4)

"this." is unnecessary?

Reply:
same for others

---
In reply to: [358414860](https://github.com/OData/WebApi/pull/1977#discussion_r358414860) [](ancestors = 358414860)